specification/refarch.adoc

```diff
@@ -3,142 +3,184 @@
-at which the memory tracking enforces confidential properties may be multiples of 
-the architectural page sizes supported by the hart MMU. Similarly, the IOMMU should 
-support a similar memory tracking lookup to enable direct device access into TVM memory 
-regions. For the CoVE reference architecture this TCB hence consists of the HW 
-(MMU, IOMMU, Memory Controller) and the SW/FW elements - TSM-driver and the TSM. 
-The TSM is responsible for enforcing isolation of confidential memory pages among TVMs 
-(via G-stage translation) - pages assigned to the TVM may belong to C | NC PMA to 
-allow for IO access. The TSM may manage additional attributes on TVM-assigned pages such as: 
-TVM-owner, Page-sub-type, TLB versioning information, Locking semaphore and additional metadata etc.
-This extended memory tracking information managed by the TSM is referred to as the EMTT.
 
-==== Handling Implicit, Explicit Accesses
-For TVM accesses for instruction fetch and page walks (virtual address 
-translation), a Confidential PMA is required to enforce the following security properties:
+==== Management of isolation for Confidential Physical Memory
+
+The SW TCB (TSM) manages the assignment of physical memory to the Confidential
+supervisor domain, while the HW TCB (hart MMU including Sv, MTT Extensions)
+enforces the access-control for confidential memory against other supervisor
+domains. The region sizes at which the memory tracking enforces isolation may
+be multiples of the architectural page sizes supported by the hart MMU.
+Similarly, the IOMMU via the IOMTT is expected to support a similar memory
+tracking lookup to enable direct IOMMU and IOMMU-managed device access into
+TVM confidential memory regions. For the CoVE reference architecture this TCB
+consists of the HW (MMU, IOMMU, Memory Controller) and the SW/FW elements -
+TSM-driver and the TSM. The TSM-driver is responsible for enforcing isolation of
+confidential memory regions (consisting of multiple pages via MTT) and the TSM
+is respondible for enforcing isolation of confidential memory pages among TVMs
+(via G-stage translation) - pages assigned to the TVM may be exclusively
+accessible to the condidential supervisor domain or may be shared with the
+hosting supervisor domain (e.g. to allow for paravirtualized IO access).
+
+[NOTE]
+====
+The TSM may manage additional attributes on TVM-assigned pages such as:
+TVM-owner, Page-sub-type, TLB versioning information, Locking semaphore and
+additional metadata etc.This extended memory tracking information managed by the
+TSM is referred to as the Extended Memory Tracking Table (EMTT).
+====
 
-* TEE Instruction fetch - security property: TVM/TSM cannot fetch code 
-from untrusted shared memory
-* TEE Paging structure walk - security property: TVM/TSM cannot locate 
-page tables in untrusted shared memory (for TVM, G-stage walks enforced 
+==== Handling Implicit, Explicit Accesses
```

Comment:
I find this section title a bit ambiguous. What about:

```suggestion
==== Handling Implicit Memory Accesses
```

Reply:
updated to "Handling Implicit & Explicit Memory Accesses"